source/common/http/filter/ext_authz.cc

```diff
@@ -0,0 +1,132 @@
+#include "common/http/filter/ext_authz.h"
+
+#include <string>
+#include <vector>
+
+#include "envoy/http/codes.h"
+
+#include "common/common/assert.h"
+#include "common/common/enum_to_int.h"
+#include "common/ext_authz/ext_authz_impl.h"
+#include "common/http/codes.h"
+#include "common/router/config_impl.h"
+
+#include "fmt/format.h"
+
+namespace Envoy {
+namespace Http {
+namespace ExtAuthz {
+
+namespace {
+
+static const Http::HeaderMap* getDeniedHeader() {
+  static const Http::HeaderMap* header_map = new Http::HeaderMapImpl{
+      {Http::Headers::get().Status, std::to_string(enumToInt(Code::Forbidden))}};
+  return header_map;
+}
+
+} // namespace
+
+void Filter::initiateCall(const HeaderMap& headers) {
+
+  Router::RouteConstSharedPtr route = callbacks_->route();
+  if (!route || !route->routeEntry()) {
```

Comment:
nit: route == nullptr || route->routeEntry() == nullptr

Same throughout PR.  Envoy style prefers (ptr == nullptr) to (!ptr)